frontend/public/components/resource-dropdown.tsx

```diff
@@ -99,30 +99,36 @@ const ResourceListDropdown_: React.SFC<ResourceListDropdownProps> = (props) => {
   const kinds = resources.groupBy((m) => m.kind);
   const isDup = (kind) => kinds.get(kind).size > 1;
 
+  const isKindSelected = (kind: string) => {
+    return _.includes(selected, kind);
+  };
   // Create dropdown items for each resource.
   const items = resources.map((model) => (
-    <DropdownItem key={referenceForModel(model)} model={model} showGroup={isDup(model.kind)} />
+    <DropdownItem
+      key={referenceForModel(model)}
+      model={model}
+      showGroup={isDup(model.kind)}
+      checked={isKindSelected(referenceForModel(model))}
+    />
   )) as OrderedMap<string, JSX.Element>;
-
   // Add an "All" item to the top if `showAll`.
   const allItems = (showAll
     ? OrderedMap({
-        all: (
+        All: (
```

Comment:
same question here: why the change to `All` from `all`? it doesn't look like this is the display value, just an internal string

Reply:
I need it capitalized for simplicity.